tensorflow/tools/docs/gen_cc_md.py

```diff
@@ -275,7 +275,8 @@ def get_type(self):
     return self.type
 
   def get_md_filename(self):
-    return self.get_type() + anchorize(self.get_short_name()) + '.md'
+    deftype = self.type[0].upper() + self.type[1:]
```

Comment:
@martinwicke  

:)
For the rint, I just thought that you were working on more important things.  Didn't mean to cross your commits.  
:)